scripts/irods/test/test_iticket.py

```diff
@@ -766,3 +767,86 @@ def test_write_tickets_for_collections_allow_creation_of_data_objects__issue_591
                 self.admin.run_icommand(['ichmod', '-M', 'own', self.admin.username, fname])
 
             self.admin.run_icommand(['iticket', 'delete', ticket_name])
+
+    def test_tickets_update_catalog_modify_time_when_modified__issue_7125(self):
+
+        def get_modification_and_creation_time():
+            out, err, ec = self.admin.run_icommand(['iquest', '%s...%s', "select TICKET_MODIFY_TIME, TICKET_CREATE_TIME where TICKET_STRING = '{}'".format(ticket_string)])
```

Comment:
Why is `...` necessary? Would a space not work?

If you have a valid ticket, can't we assume that both the mtime and ctime come back as non-empty strings?

Reply:
I had copied from work Alan had done earlier. A space would work.

Reply:
Ok. I was mostly just wondering. The use of `...` is fine.

It wasn't clear to me if there was some other issue that that resolves.